graph/fedschema.graphqls

```diff
@@ -0,0 +1,80 @@
+# GraphQL schema example
+#
+# https://gqlgen.com/getting-started/
+
+extend type Query {
+  """
+  Search for resources and their relationships. 
+  *[PLACEHOLDER] Results only include kubernetes resources for which the authenticated user has list permission.*
+
+  For more information see the feature spec.
+  """
+  globalSearch(input: [SearchInput]): [SearchResult]
```

Comment:
1. Do we need this `globalSearch` query or should we remove?
2. Let's remove the commented code.